README.md

```diff
@@ -1,5 +1,37 @@
 # md
 
+![summary.png](docs/images/summary.png)
+
+## Installation
+
+```sh
+$ trivy plugin install github.com/dramf/md
+```
+
+## Usage
+### Using CLI
+Formatted report will be saved to the file `trivy-report.md`
 ```sh
-$ trivy image alpine --output plugin=md
+$ trivy image alpine --output plugin=md --format json
 ```
+### Using GitHub Actions
```

Comment:
Let's add screenshot from `https://github.com/<owner>/<repository>/actions/runs/xxx` page in this section.